packages/genesys-spark-components/src/components/stable/gux-accordion/gux-accordion-section/gux-accordion-section.tsx

```diff
@@ -59,12 +59,17 @@ export class GuxAccordionSection {
   reverseHeadings: boolean = false;
 
   @Event()
-  internalsectionopened: EventEmitter<void>;
+  guxopened: EventEmitter<void>;
+
+  @Event()
+  guxclosed: EventEmitter<void>;
```

Comment:
I would +1 for `guxaccordionsectionopened`

Reply:
Also +1 for `guxaccordionsectionopened`

Reply:
+1 for `guxaccordionsectionopened`

Reply:
I think `guxopened` is fine. It aligns with the prop name and I don't believe there is any reason to be more specific with the name. Also, the longer names are hard to read.

Reply:
> I think `guxopened` is fine. It aligns with the prop name and I don't believe there is any reason to be more specific with the name. Also, the longer names are hard to read.

I might be misunderstanding this change. I assume this is making the events accessible by the consumer instead of being internal, which is why I support more specificity to prevent possible clashing events.